rig/routing_table/minimise.py

```diff
@@ -0,0 +1,112 @@
+import collections
+from rig.routing_table import MinimisationFailedError
+from rig.routing_table.remove_default_routes import minimise as \
+    remove_default_entries
+from rig.routing_table.ordered_covering import minimise as ordered_covering
+from six import iteritems
+
+
+def minimise(routing_tables, target_lengths,
```

Comment:
Perhaps renaming this as "minimise_tables" would be better to distinguish it from the "minimise" functions of each algorithm(?)